.github/actions/docker-image/action.yaml

```diff
@@ -41,6 +41,7 @@ runs:
       id: buildx
       uses: docker/setup-buildx-action@v3
 
+
```

Comment:
```suggestion

      - name: Go Build Cache for Docker
        uses: actions/cache@v4
        with:
          path: go-build-cache
          key: ${{ runner.os }}-go-build-cache-${{ hashFiles('**/go.sum') }}

      - name: Inject go-build-cache
        uses: reproducible-containers/buildkit-cache-dance@4b2444fec0c0fb9dbf175a96c094720a692ef810 # v2.1.4
        with:
          cache-source: go-build-cache
```

Reply:
@jules this block is probably needed for the buildkit dance to work. Would you verify this?